src/components/Icons/Back.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+import * as sizes from "./iconSizes";
+import { themes } from "../../theme";
+
+const backIcons = {
+  small: (color, children, props) => (
+    <svg
+      {...props}
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+    >
+      {children}
+      <g fill="none" fillRule="evenodd">
+        <path d="M16 0H0v16h16z" />
+        <path
+          fill={color}
+          fillOpacity=".5"
+          fillRule="nonzero"
+          d="M4.869 8l6.678 6.678a.774.774 0 0 1-1.094 1.095L3.227 8.547a.774.774 0 0 1 0-1.094L10.453.227a.774.774 0 0 1 1.094 1.095L4.87 8z"
+        />
+      </g>
+    </svg>
+  ),
+  large: (color, children, props) => (
+    <svg
+      {...props}
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+    >
+      {children}
+      <g fill="none" fillRule="evenodd">
+        <path d="M24 0H0v24h24z" />
+        <path
+          fill={color}
+          fillOpacity=".5"
+          fillRule="nonzero"
+          d="M17.542 22.692a.766.766 0 1 1-1.084 1.084L5.224 12.542c-.299-.3-.299-.785 0-1.084L16.458.224a.766.766 0 0 1 1.084 1.084L6.85 12l10.692 10.692z"
+        />
+      </g>
+    </svg>
+  )
+};
+
+const BackIcon = ({ size, color, children, ...props }) =>
+  backIcons[size](color, children, props);
+
+BackIcon.displayName = "BackIcon";
+
+BackIcon.defaultProps = {
+  size: sizes.smallLarge[0],
+  children: null,
+  color: themes.global.gray01
```

Comment:
Please default all colors to "currentColor" to provide styling flexibility.

Reply:
I thought the idea was to provide a default colour for most cases and allow to change it only occasionally not to set every time an icon is used.

Reply:
The current implementation the opposite. Since for every color we should have to change a prop. This can lead to us implementing state for things like `hover` & `focus`  Using `currentColor` allows us to set a color with css. Please take a look at the other icon implementations in the repo.

Reply:
I case to avoid implementing state for `hover` & `focus` `"currentColor"` can be passed as a prop.